backend/sftp/sftp.go

```diff
@@ -86,6 +87,12 @@ in the new OpenSSH format can't be used.`,
 			Help: `Optional path to public key file.
 
 Set this if you have a signed certificate you want to use for authentication.` + env.ShellExpandHelp,
+		}, {
+			Name: "knownhosts_file",
+			Help: `Optional path to known_hosts file.
+
+Set this value to enable server host key validation.` + env.ShellExpandHelp,
+			Advanced: true,
```

Comment:
Maybe we should add an OptionExample here with

```
                       Examples: []fs.OptionExample{{
                                Value: "~/.ssh/known_hosts",
                                Help:  "Use OpenSSH's known_hosts file",
                        }},
```

?

Reply:
I don't have any strong feelings, either way.  I think most people would work this out, anyway, and because of the subtle differences in implementation I'd really want them to read the docs, but I can add if you wish.

Reply:
Added